### PR TITLE
Remove Sass vars represented as CSS Properties

### PR DESCRIPTION
**Pull request by kyleshike**
This PR replaces all Sass vars with their equivalent CSS property and removes the corresponding Sass var _unless_ it is in use by a Sass mixin argument, which doesn't play well with native CSS properties.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -197,5 +197,6 @@
   "volta": {
     "node": "20.12.1",
     "yarn": "4.1.1"
-  }
+  },
+  "packageManager": "pnpm@9.0.4+sha1.b198ac6d38244fd829253720f9daafd6a606834d"
```
Comment (kyleshike) [inline package.json:201]:
```suggestion
  }
```

File: scss/forms/form_group.scss [hunk 2]
```diff
@@ -10,8 +9,8 @@
   margin: 8px 0;
 
   &--bordered {
-    border: 1px solid $ux-gray-400;
-    border-radius: $ux-border-radius;
+    border: 1px solid var(--ux-gray-400);
+    border-radius: 4px;
```
Comment (kyleshike) [inline scss/forms/form_group.scss:13]:
```suggestion
    border-radius: var(--ux-border-radius);
```

File: src/Table/Table.mdx
Comment (kyleshike) [inline src/Table/Table.mdx:1]: not quite sure how this got in, but this is a result of autoformatting in accordance with md styleguides

File: scss/colors/palette.scss [hunk 3]
```diff
@@ -1,146 +1,39 @@
-// When updating this file, please export the color variable in variables.scss
-// so it is available in src/Styles/palette.js
+// DEPRECATED -- use css_properties.scss
+// The only reason these variables still exist is because they are currently in use by Sass mixins.
+// They can be iteratively removed as we continue to kill Sass.
```
Comment (jasonbasuil) [inline scss/colors/palette.scss:3]: 👍

**Review by jasonbasuil — APPROVED** [review]
Seems straightforward enough 🚀

**Review by gabescarbrough — APPROVED** [review]
🔥 

Not blocking to this PR at all but we should discuss which `ux` variables are supported by Synthesis (or need an equivalent) and move them over to the `synth` prefix

**Comment by kyleshike** [comment]
> Not blocking to this PR at all but we should discuss which ux variables are supported by Synthesis (or need an equivalent) and move them over to the synth prefix

100%.  I feel like this can be one of the things we tackle next week

**Pull request merged**